npm/react/examples/react-scripts-folder/cypress.config.js

```diff
@@ -5,4 +5,17 @@ module.exports = {
   'viewportWidth': 500,
   'viewportHeight': 800,
   'componentFolder': 'cypress/component',
+  'component': {
+    setupNodeEvents (on, config) {
+      // load file devServer that comes with this plugin
+      // https://github.com/bahmutov/cypress-react-unit-test#install
+      const devServer = require('@cypress/react/plugins/react-scripts')
```

Comment:
This should be specified as the `devServer` property on `'component'`, right? Or was that going to be in a separate step?

Reply:
Yeah, I was thinking to achieve it on a separate PR - so it can be easier to review, my thoughts were make it work as it is right now, then add the new functionality for it